machine_test.go

```diff
@@ -781,6 +801,18 @@ func TestCaptureFifoToFile(t *testing.T) {
 	}
 	defer os.Remove(fifoPath)
 
+	fifoLogPath := fifoPath + ".log"
+	fifo, err := os.OpenFile(fifoLogPath, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0644)
```

Comment:
This isn't a fifo and shouldn't be called `fifo`.  This is your destination file, so call it `destination` or something like that.